spec/controllers/ctc/questions/irs_language_preference_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+describe Ctc::Questions::IrsLanguagePreferenceController do
+  let(:irs_language_preference) { nil }
+  let(:intake) { create :ctc_intake, irs_language_preference: irs_language_preference }
+
+  describe "#edit" do
+    before do
+      sign_in intake.client
+    end
+
+    render_views
+
+    context "locale is es and irs_language_preference is nil" do
+      it "Spanish is selected" do
```

Comment:
I suggest `it "selects Spanish by default"` so that it flows like an English sentence.

Reply:
Per Tuple conversation, it reads OK when you see the whole RSpec test in context, but we still ended up changing it.